src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+.page {
+  font-family: $main-font-family;
```

Comment:
Ensure that the variable `$main-font-family` is defined in your SCSS files. If it's missing, this could lead to compilation errors.